lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -269,7 +260,7 @@ def test_text3d():
     ax.set_zlabel('Z axis')
 
 
-@image_comparison(baseline_images=['trisurf3d'], remove_text=True, tol=0.03)
+@image_comparison_png(baseline_images=['trisurf3d'], remove_text=True, tol=0.03)
```

Comment:
`E501 line too long (80 > 79 characters)`, here and in 5 other places

Reply:
Could fix by renaming to `png_comparison`

Reply:
indeed... should be fixed now